roles/cadvisor/molecule/alternative/prepare.yml

```
@@ -0,0 +1,24 @@
+---
+- name: Run local preparation
+  hosts: localhost
```

Perhaps we need something like

```suggestion
  hosts: localhost
  vars_files: 
    - ../defaults/main.yml
```

To make `cadvisor_binary_url` variable available

---

I put this back to the original pattern, which is what we have in all of the other role tests.

---

Right ok, we'll put this on the back burner for all the roles then